src/main/java/com/mastery/java/task/dto/Employee.java

```diff
@@ -1,7 +1,85 @@
 package com.mastery.java.task.dto;
 
+import lombok.Builder;
+import lombok.ToString;
+
+import javax.persistence.Entity;
+import java.util.Date;
+
+@Entity
+@Builder
+@ToString
 public class Employee {
-    private Long employeeId;
+
+    private int employeeId;
     private String firstName;
+    private String lastName;
+    private int departamentId;
+    private String jobTitle;
     private Gender gender;
+    private Date dateOfBirth;
+
+    public Employee() {
+    }
+
+    public Employee(int employeeId, String firstName, String lastName, int departamentId, String jobTitle, Gender gender, Date dateOfBirth) {
+    }
+
+    public int getEmployeeId() {
+        return employeeId;
+    }
```

Comment:
please use Lombok @Data @AllArgsConstructor and @NoArgsConstructor instead getters, setters and constructors.